homeassistant/components/api_streams/sensor.py

```diff
@@ -88,3 +46,13 @@ def state(self):
     def unit_of_measurement(self):
         """Return the unit of measurement."""
         return "clients"
+
+    @callback
+    def _increment(self, connection):
+        self.count += 1
+        self.schedule_update_ha_state()
```

Comment:
Since this method is a `@callback`, you can call `async_schedule_update_ha_state` here, which is a `@callback` as well.